### ISSUE TITLE
test/test.hs:15:1: error: Failed to load interface for ‘Prelude.Unicode’

### ISSUE THREAD

**Issue by juhp** [roelvandijk/terminal-progress-bar#12]
```
test/test.hs:15:1: error:
    Failed to load interface for ‘Prelude.Unicode’
    It is a member of the hidden package ‘base-unicode-symbols-0.2.2.4’.
    Perhaps you need to add ‘base-unicode-symbols’ to the build-depends in your .cabal file.
```

**Comment by juhp** [comment]
Thanks!  Do you plan to do a new release on hackage?

**Comment by roelvandijk** [comment]
Sure. I'll do a point release shortly.

**Comment by roelvandijk** [comment]
It's on hackage: http://hackage.haskell.org/package/terminal-progress-bar-0.1.1

**Comment by juhp** [comment]
Thanks

Oops I think you to fix:

terminal-progress-bar-0.1.1 is out of bounds for:
- [ ] terminal-progress-bar-0.1.1 (==0.1.0). No maintainer. @roelvandijk

**Comment by roelvandijk** [comment]
Whoops! I was to quick. I'll look into it tomorrow. Thanks for checking.

**Comment by roelvandijk** [comment]
This is probably fixed by the 0.2 release.
